swiggy-pdp-cart/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import swiggy from "../../images/swiggy.png";
+import Socials from "./Socials/Socials";
+import FooterCSS from "./footer.module.css"
+export default function Footer() {
+  return (
+    <footer className={FooterCSS.footer}>
+      <div className="col-4">
+        <img src={swiggy} alt="Swiggy Logo" />
+      </div>
+      <p className="col-4">©2022 Swiggy</p>
+      <Socials />
```

Comment:
Whenever creating a group/list of elements, we always have 2 components - the list of the group component and the individual list item.

We should always name them as a pair so that it's easier to correlate the two.

Example: `SocialList` and `SocialListItem`, `SocialLinks` and `SocialLink` and so on